### DISCUSSION TITLE
How do I enhance profile page for my site?

### DISCUSSION THREAD

**Discussion by PradeepLoganathan** [adityatelange/hugo-PaperMod#753] [General]
Hello,
  Thanks for the wonderful theme and the amazing community being created here. I would like to add a featured posts section to my profile page. Should I add a new layouts/partial/... page? What is the right way to do it? Also, should I be aware of any pitfalls of doing this?

**Comment by RoneoOrg** [comment]
Hi,

What are you missing with [the provided layouts](https://github.com/adityatelange/hugo-PaperMod/wiki/Features#regular-mode-default-mode)?
Be sure to check out the ["pin post" feature](https://github.com/adityatelange/hugo-PaperMod/wiki/FAQs#pin-a-post)

Regards
